tests/test_cargo_build_lib.rs

```diff
@@ -52,3 +53,16 @@ test!(build_with_no_lib {
                 execs().with_status(101)
                        .with_stderr("no library targets found"));
 });
+
+test!(build_with_relative_cargo_home_path {
+    let p = project("foo")
+        .file("Cargo.toml", &basic_bin_manifest("foo"))
+        .file("src/main.rs", r#"
+            fn main() {}
+        "#);
+        env::set_var("CARGO_HOME", "./cargo_home/");
```

Comment:
@alexcrichton Ah, I would have to add a dependency, like here:

https://github.com/rust-lang/cargo/issues/1655